src/styles/_header.scss

```diff
@@ -37,5 +37,37 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: #00acdc;
+      cursor: pointer;
+
+      &::after {
+        content: "";
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        height: 4px;
+        width: 100%;
+        background: #00acdc;
+        border-radius: 8px;
+        transition-duration: 300ms;
+        animation-name: move;
+        animation-duration: 300ms;
+      }
+    }
+
+  }
+}
+
+@keyframes move {
+
+  from {
+    transform: scaleX(0);
+  }
+
+  to {
+    transform: scaleX(1);
   }
```

Comment:
It's redundant, use only correct transition for your card and change transform property on hover state